server/server.go

```diff
@@ -258,20 +227,27 @@ func (s *ArgoRolloutsServer) WatchRolloutInfo(q *rollout.RolloutInfoQuery, ws ro
 	return nil
 }
 
-func (s *ArgoRolloutsServer) ListReplicaSetsAndPods(ctx context.Context) ([]*appsv1.ReplicaSet, []*corev1.Pod, error) {
-	s.NamespaceVC.Start(ctx)
```

Comment:
Looks like `NamespaceVC` can be removed from the `type ArgoRolloutsServer struct` since we use the KubeClientset.

Reply:
Yes, it's removed from `ArgoRolloutsServer`. I guess you're looking at old file

Reply:
Hi, @perenesenko , sorry, I forgot to refer to the field in the struct
https://github.com/argoproj/argo-rollouts/blob/96ba0302f4d904e9aba36c3341025f4f51eaed81/server/server.go#L73

With the change in this PR, `NamespaceVC` seems being used in nowhere, so it could be deleted from the struct.

Reply:
@huikang see this message: https://github.com/argoproj/argo-rollouts/pull/1291/files#r655771666

Reply:
Got it. Thanks